index.html

```diff
@@ -15,36 +15,32 @@
 
     <!--Header-->
 
-    <header class="header">
+    <header class="header header-mobile">
             <a class="header-text" href="#">Welcome</a>
-                <nav>
-                    <label for="trigger">
-                        <input type="checkbox" class="trigger" id="trigger" />
-
-                        <div class="hamburger">
-                            <span></span>
-                            <span></span>
-                            <span></span>
-                        </div>
-
-                        <ul class="navi">
-                            <li><a class="color-text" href="#project-section">Portfolio</a></li>
-                            <li><a class="color-text" href="#about-section">About</a></li>
-                            <li><a class="color-text" href="#contact-section">Contact</a></li>
-                        </ul>
-                    </label>
-                </nav>     
+            <nav>
+                <div class="hamburger">
+                    <span></span>
+                    <span></span>
+                    <span></span>
+                </div>
+
+                <ul class="navi">
+                    <li><a class="color-text cancel" href="#project-section">Portfolio</a></li>
+                    <li><a class="color-text cancel1" href="#about-section">About</a></li>
+                    <li><a class="color-text cancel2" href="#contact-section">Contact</a></li>
+                </ul>
+            </nav>     
```

Comment:
- [ ] [REQUIRED] You almost met the requirements mentioned by the previous code reviewer 👍, but it looks like you didn't fully meet them, don't worry, it's very easy to fix them, I trust you to fix them 😉. When the `mobile menu` is `scrolled down`, the open areas at the `top` and `bottom` of the mobile menu container still do not disappear, and even when there is `no scroll`, the body elements are visible at the bottom of the `menu` and the `close button` disappears. Please solve these problems 😃 

![image](https://user-images.githubusercontent.com/63915024/195400167-cbb0cfbb-7292-4000-a140-f68959f5a86d.png)